test/test_prototype_transforms_consistency.py

```diff
@@ -655,37 +655,58 @@ def test_call_consistency(config, args_kwargs):
     )
 
 
-@pytest.mark.parametrize(
-    "config",
-    [config for config in CONSISTENCY_CONFIGS if hasattr(config.legacy_cls, "get_params")],
-    ids=lambda config: config.legacy_cls.__name__,
+get_paramsl_parametrization = pytest.mark.parametrize(
```

Comment:
Not sure if this is a typo

```suggestion
get_params_parametrization = pytest.mark.parametrize(
```